plugins/remote-config/tests/ab_rc-method.js

```diff
@@ -0,0 +1,363 @@
+const spt = require('supertest');
+const should = require('should');
+const testUtils = require('../../../test/testUtils');
+
+const request = spt(testUtils.url);
+let API_KEY_ADMIN = testUtils.get("API_KEY_ADMIN");
+let APP_KEY = testUtils.get('APP_KEY');
+let APP_ID = testUtils.get("APP_ID");
+let countlyDb;
+
+describe('ab and rc Method', () => {
+
+    before(async() => {
+        await request
+            .post('/i/remote-config/add-parameter')
+            .send({
+                api_key: API_KEY_ADMIN,
+                app_id: APP_ID,
+                app_key: APP_KEY,
+                parameter: JSON.stringify({
+                    parameter_key: 'normal_param',
+                    default_value: 1,
+                    description: '-',
+                    conditions: [],
+                    status: 'Running',
+                    expiry_dttm: null,
+                }),
+            })
+            .expect('Content-Type', /json/).expect(200);
+
+        await request
+            .post('/i/remote-config/add-parameter')
+            .send({
+                api_key: API_KEY_ADMIN,
+                app_id: APP_ID,
+                app_key: APP_KEY,
+                parameter: JSON.stringify({
+                    parameter_key: 'ab_param',
+                    default_value: 10,
+                    description: '-',
+                    conditions: [],
+                    status: 'Running',
+                    expiry_dttm: null,
+                }),
+            })
+            .expect('Content-Type', /json/).expect(200);
+
+        await request
+            .post('/i/remote-config/add-parameter')
+            .send({
+                api_key: API_KEY_ADMIN,
+                app_id: APP_ID,
+                app_key: APP_KEY,
+                parameter: JSON.stringify({
+                    parameter_key: 'ab_param_2',
+                    default_value: 11,
+                    description: '-',
+                    conditions: [],
+                    status: 'Running',
+                    expiry_dttm: null,
+                }),
+            })
+            .expect('Content-Type', /json/).expect(200);
+
+        await request
+            .get('/i/ab-testing/add-experiment')
+            .query({
+                api_key: API_KEY_ADMIN,
+                app_id: APP_ID,
+                app_key: APP_KEY,
+                experiment: JSON.stringify({
+                    name: 'fetch_remote_config_ab_testing',
```

Comment:
Can this experiment be named specific to this test file?
It's to avoid clashing with the test for fetch remote config endpoints